src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public Figure getRandomFigure() {
+        int index = new Random().nextInt(5);
```

Comment:
no need create Random class for each call, once is enough. Put it in a class field and use it everywhere